packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -3391,7 +3497,7 @@ The ID of a resource is unique to the Resource Set, whereas the link that points
 
 ### Binding object
 
-A Binding represents the assignment of a [Custom Role](#custom-role-object) to a list of admins. Bindings are specific to a [Resource Set](#resource-set-object). There is a maximum of one Binding object per Role in a Resource Set.
+A Binding represents the assignment of a [Custom Role](#custom-role-object) to a list of admins. Bindings are specific to a [Resource Set](#resource-set-object). There’s a maximum of one Binding object per Role in a Resource Set.
```

Comment:
nit: Can we keep it as `There is`?

Reply:
Our style guide is pretty clear about using contractions to convey a more relaxed tone, especially when dealing with technical subjects. It should stay as it is.